packages/dapp/src/components/utility/TransactionStatusModalsHOC.tsx

```diff
@@ -153,10 +155,17 @@ export const hasTransactionStatusModals = (transactionStatusModalConfig: Transac
         (multiStepTransactionLabels && multiStepTransactionLabels[this.state.transactionType!]) || "1 of 1";
       const stepLabel = `Step ${stepLabelText} - ${transactionLabel}`;
       return (
-        <MetaMaskModal ipfsPost={true} waiting={true}>
+        <Modal width={558} textAlign="center">
+          <LoadingIndicator height={100} width={150} />
           <ModalStepLabel>{stepLabel}</ModalStepLabel>
           <ModalHeading>Posting to IPFS</ModalHeading>
-        </MetaMaskModal>
+          <ModalContentInsetContainer>
+            <ModalContent>This can take several minutes. Please don't close the tab.</ModalContent>
```

Comment:
we can update this to say "a few moments" or something since posting to IPFS is usually only like 10 seconds, not a blocker tho

Reply:
Sure, I’ll update this in a PR this afternoon.